scripts/wikipedia.js

```diff
@@ -0,0 +1,66 @@
+// This script adds buttons next to isbns on wikipedia pages that will redirect
+// the user to a readable digital copy of the referenced book.
+
+
+//Get all books on wikipedia page through https://archive.org/services/context/books?url=...
+function getJSON(){
+  let url = "https://archive.org/services/context/books?url=" + location.href;
+  fetch(url)
+    .then(res => res.json())
+    .then(data => {
+      let books = document.getElementsByClassName('citation book');
+      for(let book of books){
+        let isbn = getISBNFromCitation(book);
+        let id = getIdentifierFromISBN(isbn, data);
+        if(id){
+          let link = createLinkToArchive(id);
+          book.appendChild(link);
+        }
+      }
+    })
+    .catch(err=> console.log(err));
+}
+
+function createLinkToArchive(id){
+  let a = document.createElement('a');
+  a.className = "btn btn-primary";
+  a.href = "https://archive.org/details/" + id;
+  a.innerHTML = "Read Now";
+  return a;
+}
+
+function getIdentifierFromISBN(isbn, json){
+  if(json[isbn] && json[isbn][isbn]){
+    let id = Object.values(json[isbn][isbn]['responses'])[0]['identifier']
+    if(id){
+      return id;
+    }else{
+      return null;
+    }
+  }
```

Comment:
Please use a consistent styling.
Note that I have included `eslint` in `package.json`. Please do `npm install` and then use `eslint scripts/your-script.js` when you work on something to check code style.